src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -2027,7 +2031,7 @@ End Module").Path
             ' - update project-system to recognize the new value and pass it through
             ' - update all the tests that call this canary
             ' - update the command-line documentation (CommandLine.md)
```

Comment:
>' - update the command-line documentation (CommandLine.md) [](start = 12, length = 58)

Let's update the doc.
I think `vbc -langversion:?` will automatically pick up the new version, but may be worth a try.

Reply:
> Let's update the doc.

I am not sure what is there to update. It doesn't look like the doc lists specific versions or I am looking at a wrong document.

> I think vbc -langversion:? will automatically pick up the new version, but may be worth a try.

It does.
```
Supported language versions:
default
9
10
11
12
14
15
15.3
15.5
16 (default)
16.9 (latest)
latest
```

---
In reply to: [556982209](https://github.com/dotnet/roslyn/pull/50414#discussion_r556982209) [](ancestors = 556982209)

Reply:
Right, it looks like the bullet for CommandLine.md is obsolete too. We'll remove it next time. Thanks

---
In reply to: [558429780](https://github.com/dotnet/roslyn/pull/50414#discussion_r558429780) [](ancestors = 558429780,556982209)